lens/lens_seq.ml

```diff
@@ -12,32 +11,27 @@ let return v () = Cons (v, empty)
 let rec map f v () =
   match v () with
   | Nil -> Nil
-  | Cons (v,r) -> Cons (f v, map f r)
+  | Cons (v, r) -> Cons (f v, map f r)
 
 let rec filter f v () =
   match v () with
   | Nil -> Nil
-  | Cons (v, r) ->
-    if f v then
-      Cons (v, filter f r)
-    else
-      filter f r ()
+  | Cons (v, r) -> if f v then Cons (v, filter f r) else filter f r ()
 
 let rec filter_map f v () =
   match v () with
   | Nil -> Nil
-  | Cons (v, r) ->
-    (match f v with
-     | None -> filter_map f r ()
-     | Some v -> Cons (v, filter_map f r))
+  | Cons (v, r) -> (
```

Comment:
Consider using `begin..end` instead of brackets

Reply:
This is an ocamlformat setting which I have left to the default value as I personally prefer brackets over begin end. I find begin end quite quirky and usually results in an additional line, and auto formatting ensures correct / readable indentation.

Reply:
I personally grew up with SML's `case...of` where the only option is parentheses since SML didn't have `begin...end`.  I think using `begin...end` in OCaml is preferable since the extra space can be helpful in helping visually find the beginning or ending of a larger (like multi-screen) pattern match.  It isn't as big of an issue here.

Mental note: all future functional languages should have a built-in "end of match cases" syntax delimiter.